test/Chainweb/Test/Pact/SPV.hs

```diff
@@ -302,7 +302,7 @@ txGenerator1 time pidv sid tid = do
 
                 let pcid = Pact.ChainId $ chainIdToText sid
 
-                cmd <- mkTestExecTransactions "sender00" pcid ks "1" 10 0.01 100000 (toTxCreationTime time) txs
+                cmd <- mkTestExecTransactions "sender00" pcid ks "1" 24 0.01 100000 (toTxCreationTime time) txs
```

Comment:
Instead of 100000 let's try to size these accurately

Reply:
I think 100000  is TTLSeconds, 24 is the gas limit

Reply:
ok